docs/en/manuals/extender-docker-images.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Available Docker images
+brief: Document described available Docker images and Defold versions that used it
+---
+
+# Avaialble Docker images
+Below is a list of all avaialble Docker images in public Registry. That images can be used to run Extender in environment with old SDK that isn't supported yet.
```

Comment:
avaialble -> available

"**The** images can be ... old **SDKs** that **are no longer supported**"